src/components/admissions/AdmissionsCard.jsx

```diff
@@ -0,0 +1,37 @@
+import { Box, Image, Text } from '@chakra-ui/react';
+import React from 'react'
+import styleConstants from '../../styleConstants';
+
+const AdmissionsCard = ({imgSrc, title, content}) => {
+	return (
+		<Box
+			display={{ base: 'flex', lg: 'grid' }} 
+			gap={{base: 10, lg: 0}}
+			gridTemplateColumns='1fr 1fr'
+			flexDir={'column'}> {/* switch to horiz. on small scrns */}
+				<Image mx="auto" src={imgSrc} h="80%" alt="" />
+				<Box 
+				display={{ base: 'flex', lg: 'block' }}
+				flexDir='column'
+				alignItems='center'>
+					<Text 
+					bgColor={styleConstants.colorPrimary}
+					borderRadius={styleConstants.borderRadius}
+					color="white"
+					fontSize="xl"
```

Comment:
For all the rendered text (not just here), could you please use the font size that's on the Figma? You can click on the text in the Figma and it'll tell you it's font size in pixels